File: C++/Iniciante/1018.cpp
#include <stdio.h>
#include <iostream>

int main() {

  int valor;
  std::cin >> valor;

  printf("%d\n", valor);
  printf("%d nota(s) de R$ 100,00\n", int(valor) / 100);
  valor %= 100;
  
  printf("%d nota(s) de R$ 50,00\n", int(valor) / 50);
  valor %= 50;
  
  printf("%d nota(s) de R$ 20,00\n", int(valor) / 20);
  valor %= 20;
  
  printf("%d nota(s) de R$ 10,00\n", int(valor) / 10);
  valor %= 10;
  
  printf("%d nota(s) de R$ 5,00\n", int(valor) / 5);
  valor %= 5;
  
  printf("%d nota(s) de R$ 2,00\n", int(valor) / 2);
  valor %= 2;
  
  printf("%d nota(s) de R$ 1,00\n", valor);
    
}

File: Java/Iniciante/1018.java
import java.io.IOException;
import java.util.Scanner;

public class Main {
  
    public static void main(String[] args) throws IOException {
      Scanner scanner = new Scanner(System.in);

      int valor = scanner.nextInt();
      System.out.println(String.format("%s", valor));

      System.out.println(String.format("%s nota(s) de R$ 100,00", valor / 100));
      valor = valor % 100;

      System.out.println(String.format("%s nota(s) de R$ 50,00", valor / 50));
      valor = valor % 50;

      System.out.println(String.format("%s nota(s) de R$ 20,00", valor / 20));
      valor = valor % 20;

      System.out.println(String.format("%s nota(s) de R$ 10,00", valor / 10));
      valor = valor % 10;

      System.out.println(String.format("%s nota(s) de R$ 5,00", valor / 5));
      valor = valor % 5;

      System.out.println(String.format("%s nota(s) de R$ 2,00", valor / 2));
      valor = valor % 2;

      System.out.println(String.format("%s nota(s) de R$ 1,00", valor / 1));
    }
}

